src/plugins/index_pattern_editor/public/components/empty_prompts/empty_prompts.tsx

```diff
@@ -47,15 +47,21 @@ export const EmptyPrompts: FC<Props> = ({
   useCallback(() => {
```

Comment:
This wouldn't be a regression from this pr, but looks like this never called, hence empty prompt never uses remove cluster logic
I think the intention was to use `useMemo` or `useEffect` instead

Reply:
Agreed, lets address in a separate PR.

Reply:
https://github.com/elastic/kibana/issues/108930